src/mapbox/mapbox.ts

```diff
@@ -786,14 +786,23 @@ export default class Mapbox {
     }
   };
 
+  _uniqueLayersCount(features: MapboxGeoJSONFeature[] | null) {
+    if (!features) {
+      return 0;
+    }
+    return features.filter(
+      (feature, index) => features.findIndex(f => f.layer.id === feature.layer.id) === index
```

Comment:
This is very inefficient. Use `new Set(ids).size`

Reply:
done